hail_search/hail_search_query.py

```diff
@@ -658,6 +677,32 @@ def _format_transcript_args(self):
         })
         return args
 
+    def _get_family_passes_quality_filter(self, quality_filter, ht=None, pathogenicity=None, **kwargs):
+        passes_quality = super(VariantHailTableQuery, self)._get_family_passes_quality_filter(quality_filter)
+        clinvar_path_ht = False if passes_quality is None else self._get_clinvar_filter_ht(pathogenicity)
+        if not clinvar_path_ht:
+            return passes_quality
+
+        return lambda entries: hl.is_defined(clinvar_path_ht[ht.key]) | passes_quality(entries)
+
+    def _get_clinvar_filter_ht(self, pathogenicity):
+        if self._filter_hts.get(CLINVAR_KEY) is not None:
+            return self._filter_hts[CLINVAR_KEY]
+
+        clinvar_path_filters = self._get_clinvar_path_filters(pathogenicity)
+        if not clinvar_path_filters:
+            self._filter_hts[CLINVAR_KEY] = False
+            return False
```

Comment:
Is it possible that `self._filter_hts[CLINVAR_KEY]` is assigned a value `False` during previous calls to this function and `self._filter_hts.get(CLINVAR_KEY) is not None` becomes `True` in the subsequent calls?

Reply:
no? theres only one possible return value from this function, this is just a way of caching the result

Reply:
Okay, I see. `False` also needs to be cached, representing Clinvar is unavailable.

Reply:
I had a comment here not get posted that might make the other one make more sense:

"Is this function actually called multiple times to justify the `self._filter_hts` cache?"

Reply:
And a second question, is `clinvar_path_variants.ht` a static reference table?

Reply:
This function is called every time `_filter_entries_table` is called, which is once per project. So yes, it can be reused. And `clinvar_path_variants.ht` is a static reference table, its just the path and likely path variants without the other annotations. I realize we probably should have talked offline about the couple of pre-filter tables seqr will need, its for performance optimization so it wasn't on my radar initially